### DISCUSSION TITLE
Is it possible to turn off the bank switching/ faders scrolling on selecting channels in Nuendo

### DISCUSSION THREAD

**Discussion by RHimadiiev** [bjoluc/cubase-mcu-midiremote#48] [Q&A] [answered]
Currently when I select the channel in cubase, the control surface selects corresponding channel and scroll to it. I need the first channels visible all the time. As with MCU. Thanks!

**Comment by RHimadiiev** [comment]
Icon Qcon Pro G2

**Comment by bjoluc** [comment, marked as answer]
Yes, absolutely! Please have a look at the `enableAutoSelect` [configuration option](https://github.com/bjoluc/cubase-mcu-midiremote?tab=readme-ov-file#configuration-options) :slightly_smiling_face:
